File: 0217-contains-duplicate/0217-contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Arrays.sort(nums);
        for(int j=1;j<nums.length;j++){
            if(nums[j]==nums[j-1]){
                return true;
            }

        
        }
        return false;




    }
}

File: 0217-contains-duplicate/0217-contains-duplicate.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        arr=nums.sort()
        bool=False
        for i in range(1,len(nums)):
            if(nums[i-1]==nums[i]):
                bool=True
        return bool




        